lewis/adapters/epics.py

```diff
@@ -321,7 +321,7 @@ def _create_getter(self, func, *targets):
             raise RuntimeError(
                 'The function \'{}\' does not look like a getter function. A valid getter '
                 'function has no arguments that do not have a default. The self-argument of '
-                'methods does not count towards that number.')
+                'methods does not count towards that number.'.format(final_callable.__name__))
```

Comment:
I was intending to take a look at this tomorrow, but happy for you to go ahead now if you are both happy.